.circleci/config.yml

```diff
@@ -125,7 +125,6 @@ jobs:
           command: |
             pnpm lint:fix:all
             pnpm format:write:all
-            pnpm generate-graphql-ts-types
```

Comment:
@fhenrich33 can you add `pnpm graphql-gen` back in here? As a sanity-check in case the back-end files that the command depends on were renamed or similar

Reply:
@pylipp Done,

But maybe should we label the step something else since this isn't a style check?